Bottom: 6479ceab596af8653cf243908b5fe1a516222571
Top:    db0158a495f1a2f3bd7ba71e05a74a789d6f97c4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-22 05:22:06 +0100

Refresh of profile-noholes-dicer-and-add-

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 07f593d..da1aa32 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -89,11 +89,7 @@ hidgl_ensure_triangle_space (triangle_buffer *buffer, int count)
       exit (1);
     }
   if (count > TRIANGLE_ARRAY_SIZE - buffer->triangle_count)
-    {
-//      printf ("Ensuring we have %i triangles.. only %i out of %i available, flushing\n",
-//              count, TRIANGLE_ARRAY_SIZE - buffer->triangle_count, TRIANGLE_ARRAY_SIZE);
-      hidgl_flush_triangles (buffer);
-    }
+    hidgl_flush_triangles (buffer);
 }
 
 //static int cur_mask = -1;
